src/core/chatController.js

```diff
@@ -108,10 +108,6 @@ class ChatController {
   }
 
   getTranscript(inputArgs) {
-    if (this.connectionHelper.getStatus() === ConnectionHelperStatus.Ended) {
```

Comment:
I don't see this change in our internal ChatJS. Are we only making it here in github?

Reply:
discussed offline. This change is also being made to internal ChatJS